universal/ubi8/Dockerfile

```diff
@@ -77,7 +77,7 @@ RUN mkdir -p /home/tooling/.nvm/
 ENV NVM_DIR="/home/tooling/.nvm"
 ENV NODEJS_20_VERSION=20.7.0
 # note that 18.18.0 is the latest but 18.16.1 is the supported version downstream and in ubi8
-ENV NODEJS_18_VERSION=18.16.1
+ENV NODEJS_18_VERSION=18.17.0
```

Comment:
ubi8 provides node 18.19.1: https://catalog.redhat.com/software/containers/ubi8/nodejs-18/6278e5c078709f5277f26998?architecture=amd64&image=66042a2f85c8b87bc83e3535&container-tabs=packages

Let's bump to 18.19.1 and update the comment above

Reply:
fixed